source/sigv4.c

```diff
@@ -364,25 +365,29 @@ static void setQueryParameterValue( size_t currentParameter,
  * @param[in] keyLen The length of @p pKey.
  * @return Zero on success, all other return values are failures.
  */
-static int32_t hmacKey( HmacContext_t * pHmacContext,
-                        const char * pKey,
-                        size_t keyLen );
+static int32_t addKeyToHMAC( HmacContext_t * pHmacContext,
```

Comment:
Should this keep the `hmac` so that all three hmac functions of `completeHmac` will have the same prefix?
```suggestion
static int32_t hmacAddKey( HmacContext_t * pHmacContext,
```

Reply:
Sure